### PR TITLE
Update paths for ArcGIS.xcframework 

### PR DESCRIPTION
**Pull request by mhdostal**
Update paths for ArcGIS.xcframework as the next release of the ArcGIS SDK now builds a binary framework.

### CODE REVIEW

**Comment by mhdostal** [comment]
Includes automatic project updates for Xcode 12

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Release 1.0.7
+
+- The 100.10.0 release of the ArcGIS Runtime for iOS is now distributed as a binary framework.  This necessitated the following changes in the Maps App Xcode project file:
```
Comment (esreli) [inline RELEASE.md:3]:
```suggestion
- The 100.10 release of the ArcGIS Runtime for iOS is now distributed as a binary framework.  This necessitated the following changes in the Maps App Xcode project file:
```

File: RELEASE.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+    - The `ArcGIS.framework` framework has been replaced with `ArcGIS.xcframework`.
+    - The Build Phase which ran the `strip-frameworks.sh` shell script is no longer necessary.
+- Certification for the 100.10.0 release of the ArcGIS Runtime SDK for iOS.
```
Comment (esreli) [inline RELEASE.md:6]:
```suggestion
- Certification for the 100.10 release of the ArcGIS Runtime SDK for iOS.
```

**Review by esreli — COMMENTED** [review]
It looks like this file could use the same treatment applied in the Data Collection iOS repo.
https://github.com/Esri/data-collection-ios/pull/258#issuecomment-726925598

File: maps-app-ios.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -701,7 +711,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if [ -e $HOME/.runtime-example-apps/xcode-run-scripts/arcgis-build-number-run-script.sh ]; then\n$HOME/.runtime-example-apps/xcode-run-scripts/arcgis-build-number-run-script.sh -p ios -o $PROJECT_DIR\nelse\necho \"Cannot update ArcGIS build number.\"\nfi\n\nbash \"${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/ArcGIS.framework/strip-frameworks.sh\"\n";
+			shellScript = "if [ -e $HOME/.runtime-example-apps/xcode-run-scripts/arcgis-build-number-run-script.sh ]; then\n$HOME/.runtime-example-apps/xcode-run-scripts/arcgis-build-number-run-script.sh -p ios -o $PROJECT_DIR\nelse\necho \"Cannot update ArcGIS build number.\"\nfi\n";
```
Comment (esreli) [inline maps-app-ios.xcodeproj/project.pbxproj:714]: Would you get rid of the second build script, like you did for Data Collection?

**Review by esreli — CHANGES_REQUESTED** [review]

**Review by esreli — CHANGES_REQUESTED** [review]
Would you update the app marketing version to match the release version?

**Review by esreli — APPROVED** [review]

**Pull request merged**